server.js

```diff
@@ -18,9 +33,36 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// Get the 20 most recent thoughts
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+// Send a new thought
+app.post('/', async (req, res) => {
+  const { message } = req.body
+  const thought = new Thought({
+    message: message,
+  })
```

Comment:
When the key and value are the same, you could write like this:
`const thought = new Thought({ message })`